### PR TITLE
Added fonts

### PR DESCRIPTION
**Pull request by ovekyc**

### CODE REVIEW

File: app/view/app-component.js [hunk 1]
```diff
@@ -36,6 +38,20 @@ export default class AppComponent extends React.Component {
     this.setState({appSetting: newSetting});
   }
 
+  setSourceLanguage(lang) {
+    ipcRenderer.send('change-source-lang', lang);
+    let newSetting = this.state.appSetting;
```
Comment (ovekyc) [inline app/view/app-component.js]: You may use const here

File: app/view/app-component.js [hunk 2]
```diff
@@ -36,6 +38,20 @@ export default class AppComponent extends React.Component {
...
+    newSetting.sourceLanguage = lang;
+    this.setState({appSetting: newSetting});
+  }
+
+  setTargetLanguage(lang) {
+    ipcRenderer.send('change-target-lang', lang);
+    let newSetting = this.state.appSetting;
```
Comment (ovekyc) [inline app/view/app-component.js]: ditto

File: app/view/settingView.js [hunk 3]
```diff
@@ -1,11 +1,7 @@
 import React from 'react';
 import autobind from 'autobind-decorator';
 import Select from 'react-select';
-
-const options = [
-    { value: 'one', label: 'One' },
-    { value: 'two', label: 'Two' }
-];
+import lang from './lang';
```
Comment (ovekyc) [inline app/view/settingView.js]: please move lang.json to resource directory

File: app/view/settingView.js [hunk 4]
```diff
@@ -75,8 +71,9 @@ export default class SettingView extends React.Component {
             <div style={{flexBasis: 33}}/>
             <div style={{flexBasis: 128}}>
               <Select
-                options={options}
-                onChange={()=>{}}
+                options={lang.languages}
+                onChange={(val)=>this.props.setSourceLanguage(val)}
```
Comment (ovekyc) [inline app/view/settingView.js]: Couldnt we just code it like 
`{this.props.setSourceLanguage}`
If it does not work, please let me know. Im not sure right now

File: app/view/settingView.js [hunk 5]
```diff
@@ -88,8 +85,9 @@ export default class SettingView extends React.Component {
             <div style={{flexBasis: 33}}/>
             <div style={{flexBasis: 128}}>
               <Select
-                options={options}
-                onChange={()=>{}}
+                options={lang.languages}
+                onChange={(val)=>this.props.setTargetLanguage(val)}
```
Comment (ovekyc) [inline app/view/settingView.js]: ditto

**Review by yuda1124 — APPROVED** [review]

**Pull request merged**